EIPS/eip-draft_revert_opcode.md

```diff
@@ -0,0 +1,47 @@
+## Preamble
+
+    EIP: <to be assigned>
```

Comment:
You can either assign the issue or PR number. In this case it would be better to assign the issue # since that is what it has been referred to most widely.